packages/vuetify/src/components/VCalendar/VCalendarCategory.ts

```diff
@@ -44,14 +53,16 @@ export default VCalendarDaily.extend({
         week: this.days, ...day, index,
       }
 
-      const children = this.parsedCategories.map(category => this.genDayHeaderCategory(day, this.getCategoryScope(scope, category)))
+      const children = this.parsedCategories.map(category => {
+        return this.genDayHeaderCategory(day, this.getCategoryScope(scope, category))
+      })
 
       return [this.$createElement('div', data, children)]
     },
-    getCategoryScope (scope: any, category: string) {
+    getCategoryScope (scope: any, category: any) {
```

Comment:
```suggestion
    getCategoryScope (scope: any, category: { categoryName: string }) {
```

Reply:
typing should be fixed in similar manner in other places too, also why not passing just the string instead of the object with `categoryName` prop?